extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/MSQControllerTask.java

```diff
@@ -263,4 +265,9 @@ public static boolean isIngestion(final MSQSpec querySpec)
   {
     return querySpec.getDestination() instanceof DataSourceMSQDestination;
   }
+
+  public static boolean writeResultsToDurableStorage(final MSQSpec querySpec)
```

Comment:
nit: Naming. From the name the method seems like it writes the results to durable storage
```suggestion
  public static boolean isWriteResultsToDurableStorage(final MSQSpec querySpec)
```